src/components/transactions/modals/ClaimWRBTCModal.vue

```diff
@@ -178,10 +202,10 @@ export default {
     getDataTypeObject(nonce) {
       return {
         domain: {
-          chainId: this.transaction.destinationChainId,
           name: 'RSK Token Bridge',
-          verifyingContract: this.sharedState.currentConfig.bridge.toLowerCase(),
           version: '1',
+          chainId: this.transaction.destinationChainId,
+          verifyingContract: this.sharedState.currentConfig.bridge,
```

Comment:
Good catch! If this value needs to be in Ethereum Checksum maybe we should convert that to avoid this failing in the feature for copying with the RSK checksum or with lower case